tests/FakeItEasy.Specs.netstd/ICloneable.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace FakeItEasy.Specs
+{
+    public interface ICloneable

```

Comment:
An alternative would be to just use a different interface in `CreationOptionsSpecs.cs` and `FakeOptionsBuilderSpecs.cs`, but this is good.
We could always change interfaces later, if we're so inclined.